lib/Doctrine/ORM/Tools/Pagination/LimitSubqueryWalker.php

```diff
@@ -157,20 +159,20 @@ private function validate(SelectStatement $AST)
     /**
      * Retrieve either an IdentityFunction (IDENTITY(u.assoc)) or a state field (u.name).
      *
-     * @param \Doctrine\ORM\Query\AST\PathExpression $pathExpression
+     * @param mixed $expression
```

Comment:
This should not be `mixed`: let's try avoiding `mixed` overall

Reply:
I can replace it by `Doctrine\ORM\Query\AST\Node`?

Reply:
That would be much better, thanks